app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -92,19 +91,17 @@ class PatientRepository @Inject constructor(
           }
         }
         .doOnNext { timingTracker.start(sortByVisitedFacilityAnalytics) }
-        .compose(sortByCurrentFacility())
+        .compose(sortByCurrentFacility(loggedInUser))
```

Comment:
I can see that we need `loggedInUser` here to only get the current facility eventually.
Why can't we pass the current facility instead then? 
That would also add more sense to the method called `sortByCurrentFacility` which would obviously need the current facility.